graphql/util.go

```diff
@@ -514,6 +505,23 @@ func removeGeneralSubscriptions(usr *user.DBUser, subscriptions []event.Subscrip
 	return filteredSubscriptions
 }
 
+func makePatchDuration(t, m string) *PatchDuration {
```

Comment:
Despite this being a helper function, t and m aren't helpful variables with understanding what's supposed to be passed in here -- can you elaborate a bit here?

Reply:
(Also not seeing where this is used)

Reply:
It's used in graphql/patch_resolver.go. 
renamed for clarity

Reply:
Ah yes I didn't see to expand this file ,thanks!